lib/devise/passkeys.rb

```diff
@@ -11,6 +11,7 @@
 require_relative "passkeys/version"
 
 module Devise
+  # All Passkeys sub-modules, models, classes, concerns etc are inherited from here.
```

Comment:
@tcannonfodder : what do you think of a module doc like this:
```
  # Devise extension to use passkeys instead of passwords
  #    Passkeys are the new standard to authenticate on the web: a safer and easier replacement for passwords
  #    All sub-modules, models, classes, concerns etc are inherited from here
```

The second line is from www.passkeys.com.
We could also add something like:
```
   #   from a technical point of view, passkeys are a public/private keypair, created per app/site (reference as Relying Party - RP)
```

Anyway, is this along the lines you are looking for this specific comment?

Reply:
That is perfect for this version of the documentation! And I like that second technical note as well!